core-services/prow/02_config/_config.yaml

```diff
@@ -345,6 +345,7 @@ plank:
       gcs_credentials_secret: gce-sa-credentials-gcs-private
   job_url_prefix_config:
     '*': https://prow.svc.ci.openshift.org/view/gcs/
+    openshift/private-ci-testing: https://deck-internal-ci.svc.ci.openshift.org/view/gcs/
```

Comment:
Does the config allow to have orgs here, too? If so, we should add `openshift-priv` here.

Reply:
Yea, since we are doing this, let's add this.

Reply:
addressed